src/extensions/expokit/expm.lisp

```diff
@@ -0,0 +1,50 @@
+;;;; expm.lisp
+;;;;
+;;;; Author: Robert Smith
+
+(in-package #:magicl-transcendental)
+
+(magicl.backends:define-compatible-no-applicable-method-behavior expm)
+(defgeneric expm (matrix)
+  (:method ((m magicl:matrix/complex-double-float))
+    (let ((ideg 6)
```

Comment:
At some point (not in this PR) we should add more comments around all the LAPACK/BLAS calls explaining what all the parameters mean as well as linking to the function in the LAPACK/BLAS documentation

Reply:
i agree, BLAS is impossible to read